Ryujinx.Graphics/Gal/OpenGL/OglTexture.cs

```diff
@@ -95,6 +95,7 @@ public void Create(long key, int size, GalImage image)
                         type,
                         IntPtr.Zero);
                     break;
+                case TextureTarget.TextureCubeMapArray:
```

Comment:
You should add this comment like in the other definitions.
```suggestion
                // Cube map arrays are just 2D texture arrays with 6 entries
                // per cube map so we can handle them in the same way
                case TextureTarget.TextureCubeMapArray:
```